0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,96 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection protocol
```

Comment:
This is still an extension protocol. It extends 1/KP with data collection-specific definitions.